src/utils/registry.ts

```diff
@@ -322,11 +345,11 @@ export class Registry {
 
   windowsExeAndDllDirectories(browserName: BrowserName): string[] {
     const browserDirectory = this.browserDirectory(browserName);
-    if (browserName === 'chromium')
+    if (browserName === 'chromium' || browserName === 'chromium-with-symbols')
       return [path.join(browserDirectory, 'chrome-win')];
-    if (browserName === 'firefox')
+    if (browserName === 'firefox' || browserName === 'firefox-beta')
       return [path.join(browserDirectory, 'firefox')];
```

Comment:
use separate dirs?

Reply:
I believe `firefox` is a correct one:
- `registryDirectory`: this is the one that has all browsers
- `browserDirectory`: this is the one that browser name and revision, e.g. `firefox-beta-1234`
- `firefox` is just inner directory inside the browserDirectory